commands/complete_func_args.py

```diff
@@ -72,3 +72,49 @@ def insert_snippet(self, data: Dict[str, Any]) -> None:
 
         template = data['template']
         active_view().run_command('insert_snippet', {'contents': template})
+
+
+class AnacondaFillFuncargs(sublime_plugin.TextCommand):
+    """Trigger parameters autocompletion with key press."""
+
+    def run(self, edit, all=False):
+
+        comp_par, comp_all = 'complete_parameters', 'complete_all_parameters'
+        anaconda_sets = sublime.load_settings('Anaconda.sublime-settings')
+
+        # no need to do anything, just run the command
+        if anaconda_sets.get(comp_all):
+            self.view.run_command("anaconda_complete_funcargs")
+            return
+
+        # temporarily enable autocompletion
+        if not anaconda_sets.get(comp_par):
+            self.view.settings().set(comp_par, True)
+        if all:
+            self.view.settings().set(comp_all, True)
+
+        self.view.run_command("anaconda_complete_funcargs")
+
+        # restore value as per settings
+        if not anaconda_sets.get(comp_par):
+            self.view.settings().set(comp_par, False)
+        if all:
+            self.view.settings().set(comp_all, False)
+
```

Comment:
The command temporarily changes settings, then restores the original value, in the meantime runs the main command.